### ISSUE TITLE
BatchNorm Folding in BNN 

### ISSUE THREAD

**Issue by Coke-das-JHL** [Xilinx/brevitas#491]
Hi!, Can you give some advice on how to apply Batch Norm Folding in BNN?

Thank you!
